src/Options.js

```diff
@@ -487,6 +505,14 @@ export function addOptions(flags, commandOptions) {
     'sourceMaps generated to file or inline with data: URL',
     (to) => { return commandOptions.sourceMaps = to; }
   );
+  flags.option('--source-root <false|string|default>',
```

Comment:
I don't really have a good sense for this. Another option might be to use true|false|&lt;string&gt;.

- `false` for no source root in the generated source map
- `true` for using basepath(output file).
- `"string"` for using the exact string as the source root.